### PR TITLE
JSUI-3062 Query summary, have a fallback string for when there is no query

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/JSUI-3062

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: unitTests/ui/QuerySummaryTest.ts [hunk 1]
```diff
@@ -307,16 +308,16 @@ export function QuerySummaryTest() {
       });
 
       it(`when the query searched is an empty string,
-          it should replace the query tag with the query searched`, () => {
+          it should put the string NoResult instead`, () => {
         test = Mock.optionsComponentSetup<QuerySummary, IQuerySummaryOptions>(QuerySummary, {
           enableNoResultsFoundMessage: true,
-          noResultsFoundMessage: `${queryTag}`
+          noResultsFoundMessage: `customMessage ${queryTag}`
```
Comment (jlblais) [inline unitTests/ui/QuerySummaryTest.ts:314]: Not sure I understand what the `customMessage` does. Can you explain? 🤔

File: strings/strings.json [hunk 2]
```diff
@@ -1578,6 +1578,11 @@
     "zh-cn": "没有找到与 {0} 相关的结果",
     "zh-tw": "沒有找到與 {0} 相關的結果"
   },
+  "noResult": {
+    "en": "No results",
+    "fr": "Aucun résultat",
```
Comment (jlblais) [inline strings/strings.json:1583]: Should be `Aucun résultats` (plural) to match the other ones.

Comment (ThibodeauJF) [reply strings/strings.json:1583]: http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=1440 😄

Comment (jlblais) [reply strings/strings.json:1583]: After all these years of french classes, I still don't know how to write 😅

Comment (ThibodeauJF) [reply strings/strings.json:1583]: 🤣🤣🤣

File: unitTests/ui/QuerySummaryTest.ts (hunk 1 above)
Comment (ThibodeauJF) [reply unitTests/ui/QuerySummaryTest.ts:313]: It's not really necessary, I saw it was used in the other tests and found it more realistic for the use case.

Comment (ThibodeauJF) [reply unitTests/ui/QuerySummaryTest.ts:313]: If the query is "test", the ouput would be "customMessage test". If the query is empty, it should default to "No result"

**Review by jlblais — APPROVED** [review]

**Review by samisayegh — APPROVED** [review]

**Review by olamothe — APPROVED** [review]

**Pull request merged**